wiki/Help_centre/Account/de.md

```diff
@@ -226,7 +226,7 @@ Ja, jeder Nutzer kann die Löschung seines Accounts durch das Senden einer E-Mai
 
 **Ja, absolut. Damit sind auch geteilte PCs wie Internetcafés oder PC Bangs gemeint.**
 
-**So lange du auf deinem eigenem Account spielst**, kannst du osu! überall spielen!
+**Solange du auf deinem eigenem Account spielst**, kannst du osu! überall spielen!
```

Comment:
Does this need to be `eigenen`? I'm bit confused myself over the *Deklinationen* of `eigen`

Reply:
Should be `eigenen` yeah
```suggestion
**Solange du auf deinem eigenen Account spielst**, kannst du osu! überall spielen!
```